### PR TITLE
Handled Summary steps

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/Form/SummaryType.php [hunk 1]
```diff
@@ -4,22 +4,24 @@
...
 
 class SummaryType extends AbstractType
 {
-    const STEP_SYNTHESIS = 'synthèse';
+    const STEP_SYNTHESIS = 'synthesis';
     const STEP_MISSION_WISHES = 'missions';
     const STEP_MOTIVATION = 'motivation';
-    const STEP_SKILLS = 'compétences';
-    const STEP_INTERESTS = 'centre d\'intérêts';
+    const STEP_SKILLS = 'competences';
```
Comment (tgalopin) [inline src/Form/SummaryType.php]: skills

File: src/Form/SummaryType.php [hunk 2]
```diff
@@ -97,20 +99,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ;
                 break;
         }
+
+        $builder->add('submit', SubmitType::class, ['label' => 'Enregistrer']);
```
Comment (tgalopin) [inline src/Form/SummaryType.php]: not sure it's useful to add the submit here

File: src/Form/SummaryType.php (hunk 1 above)
Comment (illusionOfParadise) [inline src/Form/SummaryType.php]: Will be changed in my PR to have less conflicts

File: src/Form/SummaryType.php (hunk 2 above)
Comment (HeahDude) [inline src/Form/SummaryType.php]: It's easiest this way, and this is where it belongs IMO.

**Pull request merged**